package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "purescript-ps-ast",
+  "name": "purescript-ps-cst",
   "version": "1.0.0",
-  "repository": "https://github.com/srghma/purescript-ps-ast",
+  "repository": "https://github.com/srghma/purescript-ps-cst",
```

Comment:
Maybe change repository url to https://github.com/purescript-codegen/purescript-ps-cst ??